include/GoogleSync/GoogleSync.php

```diff
@@ -810,21 +972,27 @@ public function updateSuitecrmMeetingEvent($event_local, $event_remote)
         // Create a new popup reminder for each google reminder
         foreach ($overrides as $override) {
             if ($override->getMethod() == 'popup') {
-                $sReminder = new Reminder;
+                $sReminder = BeanFactory::getBean('Reminders');
+                if (!$sReminder) {
+                    throw new Exception('Unable to get Reminder bean.');
+                }
                 $sReminder->popup = '1';
                 $sReminder->timer_popup = $override->getMinutes() * 60;
                 $sReminder->related_event_module = $event_local->module_name;
                 $sReminder->related_event_module_id = $event_local->id;
                 $reminderId = $sReminder->save(false);
 
-                $reminderInvitee = new Reminder_Invitee;
+                $reminderInvitee = BeanFactory::getBean('Reminders_Invitees');
+                if (!$reminderInvitee) {
+                    throw new Exception('Unable to get Reminders_Invitees bean.');
+                }
                 $reminderInvitee->reminder_id = $reminderId;
                 $reminderInvitee->related_invitee_module = 'Users';
                 $reminderInvitee->related_invitee_module_id = $this->workingUser->id;
                 $reminderInvitee->save(false);
             }
         }
-        return  $event_local;
+        return $event_local;
```

Comment:
nothing guarantees the $event_local given valid SuiteCRM meeting values from $event_remote getter methods. Please validate it before use it, save it or pass it as return value.

Reply:
Now checking for valid strings and start/end times.